beets/ui/commands.py

```diff
@@ -1479,17 +1479,20 @@ def move_items(lib, dest, query, copy, album, pretend, confirm=False):
             show_path_changes([(obj.path, obj.destination(basedir=dest))
                                for obj in objs])
     else:
-        if confirm:
-            objs = ui.input_select_objects(
-                u'Really %s' % act, objs,
-                lambda o: show_path_changes(
-                    [(o.path, o.destination(basedir=dest))]))
+        if export:
+            util.copy(item.path, item.destination(basedir=dest))
```

Comment:
Thanks! The next step is to look at the test code to see what it's doing, and reason about how your addition may have changed its behavior. In particular, this test is testing whether the `move` command actually moves files out of the way, and it's saying that the file that should be gone (i.e., moved way) is still there.

You can debug this by running the test locally and printing things out your code to see why the file didn't move away.